README.md

```diff
@@ -85,6 +90,38 @@ end
 
 Finally, run `pod install` to install your project's dependencies. 
 
+## Building
```

Comment:
This whole section is great, thanks for taking the time with it

Reply:
🙂